requirements.txt

```diff
@@ -6,7 +6,7 @@ python-dateutil>=2.7.5
 pytz>=2020.1
 pyarrow>=1.0.1
 chardet>=3.0.4
-fastavro>=1.0.0.post1
+fastavro>=1.0.0.post1,<1.9.0
```

Comment:
so probably don't need this anymore -- let's remove this

Reply:
I agree. Changed back